### ISSUE TITLE
airmon-ng syntax error in SVN181

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#111]
**Reported by darkAudax on 16 Feb 2007 20:38 UTC**

Here is the error message from airmon-ng aircrack-ng/aircrack-ng@:

```
airmon-ng start wlan0 6

Interface       Chipset         Driver

wifi0           Atheros         madwifi-ng
/usr/local/sbin/airmon-ng: line 231: syntax error near unexpected token `('
/usr/local/sbin/airmon-ng: line 231: `         printf " (monitor mode enabled)"'
```

It was working in aircrack-ng/aircrack-ng@6c6ef3cf83e219c6f819d3b8eb8ed7f4a4ed4773:

```
airmon-ng start wlan0 6

Interface       Chipset         Driver

wifi0           Atheros         madwifi-ng
ath0            Atheros         madwifi-ng VAP (parent: wifi0)
wlan0           RTL8187         aircrack-ng/aircrack-ng@ (monitor mode enabled)
```

**Comment by aircrack-ng** [comment]
**Modified by misterx on 16 Feb 2007 20:38 UTC**

**Comment by aircrack-ng** [comment]
**Comment by misterx on 16 Feb 2007 20:38 UTC**

(In #191) Fixed airmon-ng syntax error (Closes: #111)

**Comment by aircrack-ng** [comment]
**Modified by misterx on 16 Feb 2007 20:38 UTC**
